### ISSUE TITLE
Button to save a new task with the mouse

### ISSUE THREAD

**Issue by prkos** [johannesjo/super-productivity#3129]
### Expected Behavior

When I add a task text there should be a button that can be clicked with the mouse to save the task.

### Current Behavior

Only keyboard task save is currently available (Enter key). 

(On Linux it's easy to add text with the mouse if copying from elsewhere with highlight -> middle-click.) I have to take my hand off the mouse to press Enter on the keyboard. People who are using only mouse have no way of saving the task which is an accessibility issue.

### Steps to Reproduce (for bugs)

1. click on the "+" button 
2. add some text into the field
3. there is no way to save the task with the mouse, need to use Enter on the keyboard

**Comment by prkos** [comment]
I'm glad there is a lot of support for keyboard use. Usually it's the other way around.

But just as pressing Enter after typing text comes naturally so does pressing a button with the mouse after pasting some text with the mouse.

There is already the tip below the text field to press Enter, maybe that text can be converted to a button that does exactly that?

**Comment by Jagdfalke** [comment]
Well yes, I didn't dismiss your request entirely. But it's certainly not a bug, but a missing features. 

It is up to @johannesjo to decide what happens with this request. Although I assume he's probably not against it, but help is certainly welcome. 🙂

**Comment by prkos** [comment]
I don't know if this doesn't apply to SuperProductivity, maybe a decision was made to focus on using keyboard more or even exclusively, but generally speaking accessibility isn't a feature, it's something basic, if an app is useful to keyboard users, it will most likely be useful to mouse users as well. It's good to keep in mind that there are people who use mouse exclusively (most often due to injury).

So I don't agree accessibility issues should be labeled as feature requests, I guess they are their own category, but I also don't expect this to get a high priority, I thought it might be nice to have it documented in case it's not difficult to implement and someone might be looking for an easy task to solve. 

I'm not a programmer so I doubt I can implement it myself, but I can help with styling and wireframe mockups and workflows if there are any issues that need that sort of help.

**Comment by johannesjo** [comment]
You seem to know the subject well. Thanks for your input! Is pressing enter itself the inconvenience or is it not knowing that have to do it? The latter would be much easier to deal with.

Unfortunately it is not trivial to find a good spot for such a button within the UI without further cluttering it.

**Comment by prkos** [comment]
It's the former, pressing the Enter is an inconvenience, because I have to move my hand off the mouse to use the keyboard. It would be nice to middle-click my text into the field and then click the button to save it, all with mouse, without moving my hand between mouse and positioning it on the keyboard.

I remember seeing the instruction below the new task field to press Enter, I can't see it any more, maybe a previous version? I think that was useful when I first started the app. Anyway, such interactions should be intuitive, you shouldn't have to tell users how to use regular forms, it should be obvious from the design. 

Here is a tentative design suggestion if anyone wants to discuss this in more detail (is this called a Save button?):

- the Save button is visibly different from other buttons that toggle task options. 
- the Save button is disabled while the task field is empty, and it gets enabled when text is entered into the field (also shown with color)
- currently the description text of the field changes once you start typing (to "Create new task", which is weird, I've already figured out that is where I do it), but I'd love to have the useful example stay visible while typing 

![newtask](https://github.com/johannesjo/super-productivity/assets/200505/e7577ff3-dbab-49ec-809e-67c4b4379298)

![newtask1](https://github.com/johannesjo/super-productivity/assets/200505/f0f7bf48-b1fa-443a-99da-0e5ea63d0c1f)

Another note, possibly needs a separate Issue? The cursor changes color based on the settings for the project, but on a dark theme and dark chosen color the cursor is almost invisible, and there aren't any other visual cues that confirm I did click inside the field and can start typing. 

BTW I tried adding the task duration and estimation times through text as in the example (10m/3h) but it wouldn't work, I tried with spaces around the "/" still nothing. Is there something special I need to do to make it work?
